lib/funky/video.rb

```diff
@@ -40,6 +40,11 @@ def picture
       data[:picture]
     end
 
+    # @return [String] the name of Facebook page for the video.
+    def page_name
+      data[:from] && data[:from][:name]
```

Comment:
I simply was thinking I should convert Hash#dig to another code available before 2.3 Ruby. But I now I see we'd better have an error if they change API or something. Let me make a change to use `data.fetch(:from)[:name]`